plugins/CoreAdminHome/templates/whatIsNew.twig

```diff
@@ -25,4 +27,34 @@
 
     {% endfor %}
 
+    </div>
+
+    <button href="javascript:void 0;" class="btn whatisnew-btn whatisnew-remind-me-later" onclick="Piwik_Popover.close()"
+       title="{{ 'CoreAdminHome_WhatIsNewRemindMeLater'|translate|e('html_attr') }}">{{ 'CoreAdminHome_WhatIsNewRemindMeLater'|translate|e('html_attr') }}
+    </button>
+
+    <button href="javascript:void 0;" class="btn whatisnew-btn whatisnew-do-not-show-again" onclick="doNotShowAgain()"
+       title="{{ 'CoreAdminHome_WhatIsNewDoNotShowAgain'|translate|e('html_attr') }}">{{ 'CoreAdminHome_WhatIsNewDoNotShowAgain'|translate|e('html_attr') }}
+    </button>
+
 </div>
+
+ <script>
+     function doNotShowAgain() {
+        var ajaxRequest = new ajaxHelper();
+        ajaxRequest.addParams({module: 'API', method: 'CoreAdminHome.whatIsNewReadAll'}, 'get');
+        ajaxRequest.withTokenInUrl();
+        ajaxRequest.setFormat('json');
+        ajaxRequest.send();
```

Comment:
Does this need some kind of error handling? What if the request fails for some reason what will the user experience

Reply:
If the request fails then the dialog will still close, but the changes will not be marked as read and will be shown again (in 24hrs) which is probably the best way to handle it. If we caught the error there wouldn't really be a good place to show it and it wouldn't be helpful for the user.